AutomatedTesting/Gem/PythonTests/scripting/EditMenu_Default_UndoRedo.py

```diff
@@ -38,12 +38,12 @@ def EditMenu_Default_UndoRedo():
 
     # Preconditions
     import azlmbr.legacy.general as general
-    import editor_python_test_tools.script_canvas_tools_qt as sc_tools_qt
+    from editor_python_test_tools import QtPyO3DEEditor
```

Comment:
If you update this to from editor_python_test_tools.QtPyO3DEEditor import QtPyO3DEEditor, you can eliminate the double call on line 46.

Reply:
fixed